### ISSUE TITLE
Body is not fixed when modal appears

### ISSUE THREAD

**Issue by geneparcellano** [joturako/vue2-bootstrap-modal#7]
Thanks for putting this together, I really appreciate it. One thing I noticed, when I launch a modal, after I scroll to the bottom of the modal the parent page also begins to scroll.

The default behavior I noticed in the original bootstrap modal is that the parent page stays fixed until the modal is closed.

If there was a way to add `overflow: hidden` to the `<body>` when the modal is present, that would fix the issue.

**Comment by joturako** [comment]
Hi there,
I understand that the body changed when modal is open so that the parent stays fixed. I have decided not to make the component too complicated so I ignored that part.

If you want to mimic the default behavior then please track the state of the modal through `vuex` then appends the attributes to body when needed.

Also if I have to write it from the component part probably i will need you to pass the `body` to this component then use the variable to manipulate the body itself.

**Comment by joturako** [comment]
@geneparcellano hello, do you want to collaborate more? Which approach is better for this implementation?

**Comment by geneparcellano** [comment]
@joturako Apologies for the late reply. If there is a way to just target the main `<body>` and apply a class it would work.

I actually tried doing this

```
    methods: {
        open() {
                this.isShow = true
                this.$nextTick(function() {
                    this.isOpen = true
                    this.$refs.modal.focus()
                });
                document.body.style.overflow = 'hidden';
            },
            close() {
                this.isOpen = false
                this.$nextTick(function() {
                    setTimeout(() => {
                        this.isShow = false
                    }, 500)
                })
                document.body.style.removeAttribute('overflow');
            }
    },
```

However, it only disabled `overflow: hidden` when I click the close button. I didn't have enough time to figure out how to trigger `close()` when the user clicks outside the modal.

I'm ok with this approach, it's probably not the best but it's a fairly simple solution. Let me know what you think.

**Comment by joturako** [comment]
@geneparcellano Hi, I've fixed this in version `0.1.10` as per your instruction. When click outside modal I use the same `close` function. Let me know if this fix is ok for you.
